MenuAeropuerto.cpp

```diff
@@ -42,22 +42,93 @@ void MenuAeropuerto::selectorOpcion() {
 }
 
 void MenuAeropuerto::consultar() {
+    string claveOut;
     cout << "\n\tEntraste al consultar de Menu Aeropuerto\n";
+    cout<<"Ingrese la clave del aeropuerto para obtener los datos"<<endl;
+    cin>>claveOut;
+
+    while (!arbolAeropuertos->existe(claveOut)){
+        cout<<"Esa clave no exsiste en el programa, ingrese una que pertenezca al Diccionario de Aeropuertos"<<endl;
+        cin>>claveOut;
+    }
+
+    cout<<endl<<"El aeropuerto que quiere mostrar es: "<<endl;
+    cout<<*arbolAeropuertos->obtenerValor(claveOut);
```

Comment:
- Separar variables de símbolos
- Agregar `\t` para indentar los couts
- Cambiaría el `while` por un `if/else` porque si el usuario no se acuerda la clave y la ingresa mal, o si no se cargó el archivo y el diccionario está vacio, no se puede volver para atrás. No queda otra que cerrar y volver a ejecutar el programa. Con el bloque `if/else` si ingresa una clave que no existe directamente se vuelve al menú anterior
```suggestion
void MenuAeropuerto::consultar() {
    string clave;
    cout << "\t\n\tEntraste al consultar de Menu Aeropuerto\n";
    cout << "\tIngrese el codigo IATA del aeropuerto: ";
    cin >> clave;

    if (arbolAeropuertos->existe(clave)) {
        cout << "\n\tEl aeropuerto es:\n";
        cout << *arbolAeropuertos->obtenerValor(clave);
    }
    else
        cout << "\tNo se encontro el codigo IATA " << clave << "\n";
```

Reply:
Claro